src/common/shibuyaHelper.js

```diff
@@ -1,13 +1,14 @@
 import { ApiPromise, WsProvider } from '@polkadot/api';
-import { instructionWeight, env } from './constants';
+import _ from 'lodash';
+import { instructionWeight } from './constants';
 import { getProxyAccount } from './utils';
-
-const { SHIBUYA_PARA_ID } = env;
+import { Shibuya } from '../config';
```

Comment:
Everything about ss58 and paraId should be in this `Shibuya`. The same goes to `TuringDev`.

For example, 
```
import { MangataDev, TuringDev } from '../config';
```